MOxUnit/@MOxUnitTestResult/MOxUnitTestResult.m

```diff
@@ -34,5 +34,7 @@
     s.skips=cell(0);
     s.successes=cell(0);
     s.testsRun=0;
+    s.duration=0;
+    s.xml_reports=cell(0);
```

Comment:
- this would add some duplication in the information represented.
- using this approach, XML results are not updated after each test (as is currently the case) but only after all tests have been run.